index.js

```diff
@@ -1 +1,33 @@
-// code away!
+// code away
+const express= require('express')
+const helmet= require('helmet')
+const server= require('./server')
+const userRouter= require('./users/userRouter')
+const postRouter= require('./posts/postRouter')
+
+
+server.use(helmet())
+
+server.use(express.json())
+
+
+server.use('/users', userRouter)
+server.use('/posts', postRouter)
+
+server.use((req, res) => {
+    res.status(404).json({
+        message: "Route not found"
+    })
+})
+
+server.use((err, req, res, next) => {
```

Comment:
I see that you set up the error middleware, but I don't see you using that last one?